test/helpers/mentor_dashboard_helper_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
+
+class MentorDashboardHelperTest < ActionView::TestCase
+  test 'time_ago_in_days calculates time in days' do
+    two_days_ago = Date.today - 38.days
+    assert_equal '38 days', time_ago_in_days(two_days_ago)
+  end
+
+  test 'time_ago_in_days calculates time less than one day in hours or minutes' do
+    two_hours_ago = (Time.now - 2.hours).to_datetime
+    assert_equal time_ago_in_words(two_hours_ago), time_ago_in_days(two_hours_ago)
```

Comment:
This is very confusing. Why is it called `time_ago_in_days` if it will use hours or minutes too?
Where is the time_ago_in_words method from? Why is the assertion defined as being the same as the words method? How would the reader know that the words method is doing the right thing?

Generally test assertions should be against hard-coded, simple values, not against another implementation, unless the test is actually testing that two implementations are the same as part of a long-term refactoring, or something similar.

As a general rule, I would avoid using local variables if the name basically says the same thing as the implementation. `two_hours_ago` is no more readable than `Time.now - 2.hours`. That also avoids the problem that you have above when you end up with `two_days_ago = Date.today - 38.days` because you changed your mind about something. If you would have to change the name of the variable if you changed something trivial about the expression that you used to define it, that's typically a hint that the variable name is too specific or too concrete.